webapp-django/crashstats/settings/base.py

```diff
@@ -715,6 +718,16 @@ def path(*dirs):
             'virtual_host': config('RABBITMQ_VIRTUAL_HOST', '/'),
             'port': config('RABBITMQ_PORT', 5672),
         },
+        'boto': {
+            'access_key': config('resource.boto.access_key', ''),
+            'bucket_name': config(
+                'resource.boto.bucket_name', 'crashstats'),
+            'prefix': config('resource.boto.prefix', ''),
+            'keybuilder_class': config(
+                'resource.boto.keybuilder_class',
+                'socorro.external.boto.connection_context.DatePrefixKeyBuilder'
+            ),
+        }
```

Comment:
Hmm... Now I'm getting tired but I thought my changes were so that nothing needs to be configured once this PR lands. ...because we use `socorro/common` as well as `socorro/webapp`.